events/tests/importers/test_kulke.py

```diff
@@ -269,10 +269,18 @@ def test__handle_removed_events_superevent(self):
             begin_date=now - timedelta(days=60),
         )
 
-        self.assertFalse(Event.objects.filter(id=super_1_event_1.id).exists())
-        self.assertFalse(Event.objects.filter(id=super_1_event_2.id).exists())
-        self.assertFalse(Event.objects.filter(id=super_1.id).exists())
-        self.assertTrue(Event.objects.filter(id=super_2_event_1.id).exists())
-        self.assertTrue(Event.objects.filter(id=super_2_event_2.id).exists())
-        self.assertTrue(Event.objects.filter(id=super_2.id).exists())
-        self.assertFalse(Event.objects.filter(id=super_3.id).exists())
+        self.assertFalse(
+            Event.objects.filter(id=super_1_event_1.id, deleted=False).exists()
+        )
+        self.assertFalse(
+            Event.objects.filter(id=super_1_event_2.id, deleted=False).exists()
+        )
+        self.assertFalse(Event.objects.filter(id=super_1.id, deleted=False).exists())
+        self.assertTrue(
+            Event.objects.filter(id=super_2_event_1.id, deleted=False).exists()
+        )
+        self.assertTrue(
+            Event.objects.filter(id=super_2_event_2.id, deleted=False).exists()
+        )
+        self.assertTrue(Event.objects.filter(id=super_2.id, deleted=False).exists())
+        self.assertFalse(Event.objects.filter(id=super_3.id, deleted=False).exists())
```

Comment:
Same here, if there's any soft deleted events, maybe we could test that `self.assertTrue(Events.objects.filter(..., deleted=True).exists())`